Dockerfile

```diff
@@ -1,22 +1,29 @@
 # build environment
 FROM node:10.8.0 as builder
+
 RUN mkdir -p /usr/omniboard
 WORKDIR /usr/omniboard
-ENV PATH /usr/omniboard/node_modules/.bin:$PATH
-
-# grab tini for signal processing and zombie killing
-ENV TINI_VERSION v0.18.0
-ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
-RUN chmod +x /tini
-# Having "--" at the end will enable passing command line args to npm script
-ENTRYPOINT ["/tini", "--", "yarn", "run", "prod"]
+ENV PATH=/usr/omniboard/node_modules/.bin:$PATH GENERATE_SOURCEMAP=false
 
 # install yarn
 RUN npm install -g yarn
 
 COPY . /usr/omniboard
 
-WORKDIR /usr/omniboard
 RUN yarn install
 
 EXPOSE 9000
+
+FROM node:10-alpine
+
+WORKDIR /usr/omniboard
+
+# Having "--" at the end will enable passing command line args to npm script
+ENTRYPOINT ["npm", "run", "prod", "--"]
+
+ENV PATH /usr/omniboard/node_modules/.bin:$PATH
+
+COPY --from=builder /usr/omniboard/package.json /usr/omniboard/package.json
+COPY --from=builder /usr/omniboard/dist /usr/omniboard/dist
+COPY --from=builder /usr/omniboard/web/build /usr/omniboard/web/build
+RUN npm install --production && npm cache clean --force
```

Comment:
Can we use yarn here as well?

Reply:
We can, and it might be yarn is actually available already in the node:10-alpine base image, but I only noticed that might be the case after I made the pull request. I switched to npm for this because I figured the difference while running wouldn't have any effect, but I don't know enough about npm/yarn really to say.

Reply:
Let's use yarn for consistency. In the previous step, yarn is installed by this command:
`RUN npm install -g yarn`.